dbaas-adapter/basic/role.go

```diff
@@ -84,19 +84,17 @@ func (bp BaseProvider) GetBackendRoles(roleType string) []string {
 	return []string{fmt.Sprintf(BackendRolePattern, roleType)}
 }
 
-func (bp BaseProvider) CreateRoleWithISMPermissions(enhancedSecurityPluginEnabled bool) error {
+func (bp BaseProvider) CreateRoleWithISMPermissions() error {
```

Comment:
As far as I remember we should provide here the basic ISM role, I do not see any changes here, please review.